extras/speaker-recognition/src/simple_speaker_recognition/api/routers/identification.py

```
@@ -281,48 +283,123 @@ async def diarize_identify_match(
+    if not file and not conversation_id:
+        raise HTTPException(400, "Must provide either audio file or conversation_id")
+    if file and conversation_id:
+        raise HTTPException(400, "Cannot provide both audio file and conversation_id - choose one mode")
+    if conversation_id and not backend_token:
+        raise HTTPException(400, "backend_token required when using conversation_id")
+
     # Parse transcript data
     try:
         transcript = json.loads(transcript_data)
         words = transcript.get("words", [])
         full_text = transcript.get("text", "")
```

Ruff (0.14.11)

Local variable `full_text` is assigned to but never used

Remove assignment to unused variable `full_text`

(F841)